ui/shared/InputFilter.jsx

```diff
@@ -19,6 +19,12 @@ export default class InputFilter extends React.Component {
     800,
   );
 
+  componentDidMount() {
```

Comment:
Needed this for Push Health, so I could take the `searchStr` from the URL and display the initial text in the InputFilter.  Otherwise, this filter showed blank, but the filtering was still applied.